### ISSUE TITLE
Capacities of Crossborder-Lines

### ISSUE THREAD

**Issue by MarlonSchlemminger** [openego/data_processing#290]
I did some research about the capacities for our crossborder lines (Germany to a foreign bus, not between foreign buses) and came to the conclusion that we might be overestimating the capacities to some countries, namely to Sweden und Switzerland. I found it tough to get reliable numbers, which is why I mostly want to talk about dimensions, not about precise numbers. My sources are the electricity market monitoring reports of the ACER from 2015 and 2016 (http://www.acer.europa.eu/Official_documents/Acts_of_the_Agency/Publication/ACER%20Market%20Monitoring%20Report%202015%20-%20ELECTRICITY.pdf). Both reports include tables with numbers for the thermal capacities between countries. These are my results:
![crossborder_graph](https://user-images.githubusercontent.com/22960010/41599526-cbc354d6-73d3-11e8-8493-ca5c9672faef.PNG)
![crossborder_table](https://user-images.githubusercontent.com/22960010/41599527-cbe671b4-73d3-11e8-94a8-cb3d12cf22f9.PNG)

The first column EGO are the values from our grid model and for 220kV+ only (I think the ACER report covers the transmission grid only as well) from version 0.3.2. I can't explain why there is such a large discrepancy between values of ACER 2016 and 2015 for some countries, but especially for Switzerland, our grid model seems to really overestimate the capacity. ACER values are between 10 and 15 GW, Wikipedia lists lines with a combined capacity of 12 GW and we have almost 30 GW in our grid model. The ACER did not mention lines between Germany and Sweden, but as far as I know (and Olav confirmed that), there is only a single line through the baltic sea with 600 MW, while our grid model has 3.5 GW. I checked the script where the capacity for the crossborder lines is set and it only depends on the voltage level and the amount of cables and the default for all lines is 6 cables, right? This should probably be more specific

Every other country besides Switzerland and Sweden looks acceptable to me and is generally in line with the ACER values. The Netherlands might need some confirmation though, as the one ACER value we have is almost twice as high as in our grid model as well.

In version 0.4.1, the lines to Sweden and Danmark change. The line to Sweden goes through the Danish bus now which looks kind of weird to me, because it is not like that in reality. Why did you change that, Clara?

**Comment by ClaraBuettner** [comment]
The line to Sweden has been simplified as an AC-line in the last data-models and has been changed to a DC-line (with p_nom = 600). Due to a bug, the old AC lines wasn't deleted. So the problem in Sweden will be solved soon. 
The line crossing the Sweden line was created because there's a danish bus (in otg_ehvhv_bus data there's the country_id of each bus) in the Baltic sea which needed to be connected to the center of Denmark. 
The estimation of the capacity of the lines connecting the border-crossing buses to the center is based on the number of circuits in the entso-e grid map. So each foreign bus gets a line to the center of its country with the number of circuits of the entso-e. But sometimes, our data-model splits up a line with multiple circuits into multiple lines with one circuit. In some cases, there are even different buses, like in Switzerland:
![schweizline](https://user-images.githubusercontent.com/22964326/41643890-d961ec62-746c-11e8-857a-a6df80964736.png)
 In this case, the script create a line to the center from each of the four buses with the capacity that belongs actually to the sum of all lines. I guess that's the reason for the overestimation in Switzerland.

**Comment by ulfmueller** [comment]
Checking this issue, one question popped up. @ClaraBuettner Did we ever talk about this DC line going from Germany (Bentwisch) to Denmark (Bjæverskov)? Is that a new power line? Why did we not include it?

**Comment by ClaraBuettner** [comment]
That's a good question. The line seems to be there since 1995, but we have never talked about it. I'm not sure if it is currently integrated as an AC-line (it could be the one Marlon was writing about). 
I suggest to take a closer look on this and implement it as a DC-link when I make the changes of capacities we talked about today? 

And as an update for the documentation: 
In Switzerland, the border-crossing lines from osm have a nearly similar capacity. The Entso-e is not helpful for some lines because the number of systems is given as >=3 and could be nearly everything. I just changed the capacity of some lines in Switzerland to fit to the capacity of the corresponding border-crossing lines. 
This method will be used for all foreign lines in the next change. 
I'll compare this data to the useful data of the Entso-e to check it's plausibility. 
If it's not plausible, an eTraGo function to update the capacity will be written, so multiple settings will be possible. That might be a solution for the problem when comparing our data to the ACER as well.

**Comment by MarlonSchlemminger** [comment]
The line I talked about is the second one visible in Ulfs screenshot, going from Lübeck to Sweden, not to Denmark. 
What total capacity to Switzerland do we have now after you made the changes? Thanks for your effort!

**Comment by ClaraBuettner** [comment]
I only deleted two 380kV-systems, so it's reduced by 3580 MW.

**Comment by ulfmueller** [comment]
> That's a good question. The line seems to be there since 1995, but we have never talked about it. I'm not sure if it is currently integrated as an AC-line (it could be the one Marlon was writing about).
I suggest to take a closer look on this and implement it as a DC-link when I make the changes of capacities we talked about today?

sounds good! You wrote you deleted the 2 systems to Switzerland. Did you tackle this Denmark-issue yet?

**Comment by ClaraBuettner** [comment]
Honestly, I don't understand the problem with the line to Sweden (in new versions implemented as link), can you try to explain it again? The lines cross each other but that fits to the entso-e map. 
And I just updated the electrical_neighbor script (https://github.com/openego/data_processing/tree/feature/new_neighbour_script); the lines in foreign countries now get the same capacity as the border crossing lines (which is often less then before) . In addition, I updated the line to Denmark to a DC-link.

**Comment by MarlonSchlemminger** [comment]
I evaluated the lines between electrical neighbours you added in 0.4.1. Everything seems to match the reality really good, besides DK to SE, where we have 8200 MW in the model and 3324 MW in the ACER paper. Probably a similar case to what we experienced in Switzerland?
![crossborder_foreign](https://user-images.githubusercontent.com/22960010/42561472-81447714-84f9-11e8-9cca-c0ce61950023.PNG)

**Comment by ClaraBuettner** [comment]
The capacity of lines between electrical neighbors is calculated directly with an import of the ENTSO-E  map. (in oedb: model_draft.ego_grid_pp_entsoe_line) So a problem like in Switzerland isn't possible, I guess there's a problem in the imported tables (the grid map was imported wth GridKit) since the number of circuits is higer than on the grid map online. 
I could fix this manually, but I don't know when this is solved in dp.

**Comment by MarlonSchlemminger** [comment]
Im adjusting all crossborder capacities manually myself anyways, just wanted to let you know that there might be something wrong.

**Comment by ClaraBuettner** [comment]
The lines between Sweden and Denmark are underground cables, which have less capacity the transmission lines. But the imported map dosn't include information about this, and all lines are assumed as transmission lines. That causes an overestimation and closely doubles the capacity between Denmark and Sweden. 
Thank's for the hint, I will implement this better now.

**Comment by lukasoldi** [comment]
> Im adjusting all crossborder capacities manually myself anyways, just wanted to let you know that there might be something wrong.

@MarlonSchlemminger how do you do this manual adjustment, is it a function in eTraGo?

**Comment by MarlonSchlemminger** [comment]
No, this is something I have written myself for my masterthesis. I set countrytags to each line and can then adjust the capacities percentagewise or just with flat values.
